packages/wrangler/src/index.tsx

```diff
@@ -364,6 +401,40 @@ export async function main(argv: string[]): Promise<void> {
                 "utf-8"
               )
             );
+
+            if (
+              !packageJsonContent.scripts?.start &&
+              !packageJsonContent.scripts?.deploy &&
+              shouldCreatePackageJson
+            ) {
+              await writeFile(
+                pathToPackageJson,
+                JSON.stringify(
+                  {
+                    ...packageJsonContent,
+                    scripts: {
+                      ...packageJsonContent.scripts,
+                      start: "wrangler dev src/index.js",
+                      deploy: "wrangler publish src/index.js",
+                    },
+                  },
+                  null,
+                  "  "
+                ) + "\n"
+              );
+              console.log(`To start developing on your worker, run npm start.`);
+              console.log(
+                `To publish your worker on to the internet, run npm run deploy.`
+              );
+            } else {
+              console.log(
+                `To start developing on your worker, npx wrangler dev src/index.js`
+              );
+              console.log(
+                `To publish your worker on to the internet, npx wrangler publish src/index.js`
+              );
+            }
```

Comment:
The only difference between these two large blocks of code is the entry-point path (`src/index.ts` vs `src/index.js`).

Can we move this into a helper function that takes this path as a parameter?